testData/pathfinding.data.js

```diff
@@ -0,0 +1,393 @@
+import { BuildingId } from "../src/types/main";
```

Comment:
Is it possible to have this file in `__tests__` directory? Doesn't seem like a parent directory

Reply:
We could not, because everything in __test__ will be tested. If we put it in __test__ pathfinding.data.js will be seen as a test, and it will fail.

Reply:
@RGPosadas What do you think?

Reply:
We need to then change the jest config so that it doesn't data.js or data.ts files

Reply:
Can you try ignoring the directory like in this article?
https://codewithhugo.com/jest-exclude-coverage/